src/test_util.rs

```diff
@@ -419,7 +442,10 @@ impl CountReceiver<String> {
         stream
             .take_until(tripwire)
             .map_ok(|socket| FramedRead::new(socket, LinesCodec::new()))
-            .map(|x| x.unwrap())
+            .map(|x| {
+                connected.take().map(|trigger| trigger.send(()));
+                x.unwrap()
+            })
```

Comment:
No `unwrap` for `trigger.send()` because channel can be closed to this moment? Probably `inspect` is more appropriate here.

Reply:
> No unwrap for trigger.send() because channel can be closed to this moment?

Yes. If main code doesn't care about it, then this code doesn't either.

> Probably inspect is more appropriate here.

`send` requires `self`.